packages/core/src/types.ts

```diff
@@ -246,10 +246,12 @@ export interface InitConfigRedux<TRootState = any> {
 	enhancers?: StoreEnhancer[]
 	middlewares?: Middleware[]
 	rootReducers?: ReducersMapObject<TRootState, Action>
-	combineReducers?: (
-		reducers: ReducersMapObject<TRootState, Action>
-	) => ReduxReducer<TRootState>
-	createStore?: StoreCreator
+	combineReducers?:
+		| ((
+				reducers: ReducersMapObject<TRootState, Action>
+		  ) => ReduxReducer<TRootState>)
+		| undefined
+	createStore?: StoreCreator | undefined
```

Comment:
Why we need the explicit `any`?

Reply:
Typrescript 4.4.x fails about error values used in catch methods which are not typed as any or unknown.

Reply:
Ah didn't read you correctly. Ts 4.4 fails about not compatible X | UNDEFINED to X. 


Probably it's a regression I'll take a look before merging into main